app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -212,6 +213,32 @@ class QuestionPlayerActivityTest {
     }
   }
 
+  @RunOn(TestPlatform.ROBOLECTRIC)
+  @Test
+  fun testQuestionPlayer_submitCorrectAnswer_correctTextBannerIsDisplayed() {
+    launchForSkillList(SKILL_ID_LIST).use {
+      // Option 2 is the right answer and tick icon should be visible completely
+      selectMultipleChoiceOption(optionPosition = 2)
+      onView(withId(R.id.congratulations_text_confetti_view)).check(
+        matches(
+          isCompletelyDisplayed()
+        )
+      )
+    }
+  }
+
+  @RunOn(TestPlatform.ROBOLECTRIC)
+  @Test
+  fun testQuestionPlayer_submitCorrectAnswer_confettiIsDisplayed() {
```

Comment:
landscape version for this test case.

Reply:
Done, in `QuestionPlayerActivityLocalTest`. Ditto from above about configuration changes during animation, though